project.md

```diff
@@ -11,41 +11,64 @@ _What need is met by your idea? Who are you helping? What is that person's
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+What excites me about this idea is its ability to start small lets say for the average atheletes dad recording player stats for their children, and also have the ability to grow to the level where college coaches who spend hours after a game inputing stats can do it instantly durring the game. I came up with this idea after trying to watch my brothers football games from out of state. My brother plays football in highschool in colorado, the only way I could watch his game was if they put it online. If the game wasnt online I still wanted a way to see how he did. I know my dad takes stats at every game and provides the stats to the coaches too uplaod to maxpreps which is a high school sports stats collection for all teams in the USA. But I can not call him durring the game and ask what are the team stats, and I have to wait a day for these stats to be upaloded onine. There has to be a better way and maybe a DSL could fix this problem.
+
 ### Domain
 
 _Describe the project's domain in five words._
 
+Language for real time football statistics
+
 ### Interface (syntax)
 
 _How might the user interact with the language? What does programming look
 like? Why is this the right way to interact with the problem domain?_
 
+Someone such as a parent or coach could use their phone or computer and write a sentence such as "player 1 passed for 33 yards to player 2" the DSL would know to add 33 yards to player 1's passing total and 33 yards to player 2's catching yard total. I feel like this laguage would work best using a natural laguage syntax because it would allow for the typical user such as a parent or coach to use it without having to know a laguage. Another possible way to structure this language would be to use sports terminology such as TFL(player1, num yards) which would mean that player 1 had a tackle for a loss of num yards. For coaches and parents who understand this terminology this laguage would be very easy to infer. After some more thought I belive my final syntax will be a version of a structured short hand natural language such as "P 11 20y 82" this would be short hand for "Pass player number 11 for 20 yards to player number 82". The basic structure of this shorthand notation would be (playtype/outcome)(player name or number)(number of yards)(additional player or number)(outcome). The structure would involve optional full word or short hand notation such as pass or p for playtype or John or 11 for player/number. number of yards can be positive or negative and followed by y to represent yards. And the additional player/number is only needed when it is a pass play. the optional outcome at the end can be used to mark the outcome at the end of a play such as an incomplete pass or a touchdown. Note that not all of these groups in the syntax will be used all at once each time an input is written in the dsl.
+
```

Comment:
Great use of examples, this addresses my concern above about users not knowing programming principles. I really enjoy your explanation of your thought process here, I think it would be great to allow for both the more verbose and shorthand implementations such that users could use "P 11 20y 82", "Pass player number 11 for 20 yards to player number 82", and maybe even a combination of the two such as "Pass 11 20 yards 82" for cases where users may be able to remember some shorthand definitions but not all of them.